src/commands.ts

```diff
@@ -223,10 +223,10 @@ export function registerCommands(context: vscode.ExtensionContext, prManager: IP
 		return await prManager.requestChanges(pr, message);
 	}));
 
-	context.subscriptions.push(vscode.commands.registerCommand('pr.openDescription', async (pr: IPullRequestModel) => {
-		const pullRequest = ensurePR(prManager, pr);
+	context.subscriptions.push(vscode.commands.registerCommand('pr.openDescription', async (descriptionNode: DescriptionNode) => {
+		const pullRequest = ensurePR(prManager, descriptionNode.pullRequestModel);
```

Comment:
when opening the description from the command palette or the status bar, descriptionNode can be undefined

Reply:
fixed ;)

Reply:
nice